esp/templates/survey/answers.html

```diff
@@ -59,6 +54,28 @@
                     <li>Average for all classes: {{ q.question.global_average }}</li>
                 {% endif %}
             </ul>
+            <!-- Creates a key that provides details about the Histogram Labeling-->
+            <b>Response Labels:</b>
+            {% for label in q.question.get_params.list %}
+            <ul>
+                <li> <b> {{ forloop.counter }}: </b>{{ label }} </li>
+            </ul>
+            {% endfor %}
+                {% with q.question.get_params as params %}
+                <ol style = "font-weight: bold;">
```

Comment:
Might be good to wrap the `<ol>` in an if loop that checks at least one of the lower middle or upper texts exists? Otherwise we get an empty `<ol>` that takes up a little space.